### PR TITLE
Show legal versions

### PR DESCRIPTION
**Pull request by okanji**
image

- Cleaned up the layout of the version toolbar. 
- Approved chip is disabled if not the current agreement

### CODE REVIEW

File: src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js [hunk 1]
```diff
@@ -43,6 +45,36 @@ const VersionsToolbar = ({
...
+    }
+  }))(Chip)
+
+  const createApprovedChip = current =>
+    current ? (
+      <ApprovedChip label="Approved" />
+    ) : (
+      <DisabledApprovedChip label="Approved" disabled />
+    )
+
```
Comment (calebissharp) [inline src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js:77]: Can we move these component definitions outside of the `VersionsToolbar` component? We don't want to unnecessarily re-create these functions on every render.

**Comment by micheal-w-wells** [comment]
@okanji I took Lisa off the code reviewers list, you don't have to worry about anybody other than devs on the PR, just mention the issue number here so that it's linked.  Nice work buddy : ).

When you code is merged add the 'ready to test' label'

**Comment by micheal-w-wells** [comment]
also drag it over to the 'QA' pipeline in Zenhub, forgot to mention

**Comment by okanji** [comment]
Sounds good, thanks @micheal-w-wells

File: src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js [hunk 2]
```diff
@@ -57,30 +89,51 @@ const VersionsToolbar = ({
...
+                      : moment(option.version.effectiveLegalEnd).format(
+                          'MMM DD YYYY h:mm a'
+                        )}
+                  </div>
+                }
               />
-            </MuiMenuItem>
+              {option.version.status?.name === 'Approved' &&
+                createApprovedChip(option.version.effectiveLegalEnd == null)}
+            </StyledMenuItem>
```
Comment (calebissharp) [inline src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js:136]: There are actually multiple statuses that can mean "approved", in the sense that the plan is legally binding. We're also only showing versions in this list that were legal at some point, so I think that means we'd end up showing "Approved" for every menu entry. The check you're doing will only match statuses that have the name explicitly of "Approved", but we have a couple others that *mean* approved but aren't *named* "Approved". I'm also not sure if displaying to the user that a version of plan was "approved" is important. @micheal-w-wells or @LisaMoore1, do you have any ideas on perhaps a more relevant piece of information to show here, or is this way alright?

This is an oversight on my part @okanji, I should have walked through what the statuses meant with you.

**Comment by okanji** [comment]
Resolves #704

**Comment by calebissharp** [comment]
@okanji Looks like I didn't get to this before you merged it in. Can you please open another PR to address my comments?

**Comment by okanji** [comment]
> @okanji Looks like I didn't get to this before you merged it in. Can you please open another PR to address my comments?

Yes sorry I shouldn't have merged, I'll create an issue with your comments and create a PR to resolve that.

File: src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js (hunk 1 above)
Comment (okanji) [reply src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js:77]: @calebissharp to keep you posted, I ended up removing these component definitions.

**Pull request merged**